Doc/tools/extensions/pyspecific.py

```diff
@@ -692,6 +692,10 @@ def patch_pairindextypes(app) -> None:
         # pairindextypes.pop('statement', None)
         # pairindextypes.pop('builtin', None)
 
+        # there needs to be at least one statement in this block, will be
+        # removed when the first of the below is uncommented.
```

Comment:
```suggestion
        # removed when the first of the above is uncommented.
```

Maybe also comment the stuff above to say why it's commented out?

Reply:
It's in preparation for the follow-up PRs so hopefully they won't be here for a long time. Will write a brief note though.

A

Reply:
Explanatory note added